src/main/java/pro/java/hw20/furnitureFactory/WoodenFurnitureFactory.java

```diff
@@ -0,0 +1,15 @@
+package pro.java.hw20.furnitureFactory;
+
+public class WoodenFurnitureFactory {
```

Comment:
You need to create abstract factory, something  without implementation which will be delegated to a children

Reply:
fixed, code was rewritten